index.bs

```diff
@@ -2381,17 +2381,15 @@ To <dfn>get the browsing context info</dfn> given |context|,
 </div>
 
 <div algorithm>
-To <dfn>get the descendent browsing contexts</dfn> given |parent id|, |depth|
+To <dfn>get the descendent browsing contexts</dfn> given |parent|, |depth|
```

Comment:
I think it might be sensible to just have a single parameter like we do with serialization that counts down toward 0, rather than two parameters one of which is constant and one that is incremented.

Reply:
While this might be fine I see a problem when we emit the reference to the parent browsing context. As given by the spec the `parent` should only be included for the top-most browsing context info. If we only pass-in the `maxDepth` there is no chance to actually retrieve that information anymore. Any suggestions?

Reply:
Two options I think: 1) Just always return the parent id, even when that's redundant information, or 2) have an `root` flag that's `True` for the initial call and `False` for recursive calls.

Reply:
I would choose the `root` flag to not change the data that we return and to not emit redundant information. I'll push an update soon. Thanks.